app/controllers/support/responsible_bodies_controller.rb

```diff
@@ -17,6 +17,12 @@ def show
     @users = policy_scope(@responsible_body.users).not_deleted.order('last_signed_in_at desc nulls last, updated_at desc')
     @schools = @responsible_body
       .schools
+      .gias_status_open
+      .includes(:device_allocations, :preorder_information)
+      .order(name: :asc)
+    @closed_schools = @responsible_body
+      .schools
+      .gias_status_closed
       .includes(:device_allocations, :preorder_information)
       .order(name: :asc)
```

Comment:
and as before:
```suggestion
      .gias_status_closed
      .includes(:device_allocations, :preorder_information)
      .left_outer_joins(:user_schools)
      .select('schools.*, count(user_schools.id) AS user_count')
      .group('schools.id')
      .order(name: :asc)
```

Reply:
Great spot, it only applies to the closed schools query though as it's not needed for open schools.